css-overflow-3/Overview.bs

```diff
@@ -315,6 +321,90 @@ Scrollable Overflow</h3>
 	in other coordinate systems due to transforms [[CSS3-TRANSFORMS]].
 	This means scrollbars can sometimes appear when not actually necessary.
 
+<h3 id="scrolling">
+Scrolling Overflow</h3>
+
+	A box’s [=overflow=] can be visible or clipped.
+	CSS also allows a box to be <dfn export>scroll container</dfn>
+	that allows the user to scroll clipped parts of its <a>scrollable overflow area</a>
+	into view.
+	The visual viewport of a <a>scroll container</a>
+	(through which the <a>scrollable overflow area</a> can be viewed)
+	coincides with its padding box,
+	and is called the <dfn export>scrollport</dfn>.
+	A box’s <dfn export>nearest scrollport</dfn>
+	is the [=scrollport=] of its nearest [=scroll container=] ancestor.
+
+	Scrolling operations can be initiated by the user
+	(for example, by manipulating a scrollbar, swiping a touchscreen, or using keyboard controls)
+	or by script
+	(for example, by the {{scrollIntoView()}} or {{focus()}} APIs).
+	The initial position of the [=scrollable overflow rectangle=] within the [=scrollport=]
+	before any scrolling operations take effect is the <dfn>initial scroll position</dfn>.
+	The [=initial scroll position=]
+	is typically dependent on the [=scroll container=]’s [=writing mode=],
+	and, unless otherwise specified,
+	coincides with its [=scroll origin position=].
+	However, the 'align-content' and 'justify-content' properties [[!CSS-ALIGN-3]]
+	can be used to change the [=initial scroll position=],
```

Comment:
```suggestion
	can be used to change the [=initial scroll position=];
```

Reply:
I think we usually use a comma in this pattern.